### PR TITLE
Mounting the IMDB EBS volume into the container for persistence

### PR DESCRIPTION
**Pull request by jer**
- Also switched to :LATEST tag for the container

### CODE REVIEW

File: kubernetes/api_deployment.yaml [hunk 1]
```diff
@@ -23,3 +23,7 @@ spec:
       - name: apikey
         secret:
           secretName: default-token-sw0ar
+      - name: screwdriver-imdb
+        awsElasticBlockStore:
+          volumeID: vol-3ea656b7
```
Comment (nkatzman) [inline kubernetes/api_deployment.yaml]: Where does the volumeID come from?

Comment (jer) [inline kubernetes/api_deployment.yaml]: I just manually created it in the AWS console. It's the EBS volume id.

Comment (nkatzman) [inline kubernetes/api_deployment.yaml]: Okay, it'll just require a code change if we ever delete it/recreate it

**Comment by FenrirUnbound** [comment]
👍 

dappermenacingbudgie-size_restricted

**Comment by nkatzman** [comment]
👍

**Pull request merged**